kuma/wiki/models.py

```diff
@@ -1584,7 +1598,10 @@ class DocumentZone(models.Model):
     attributes inherited by the topic hierarchy beneath it.
     """
     document = models.OneToOneField(Document, related_name='zone')
-    styles = models.TextField(null=True, blank=True)
+    css_filename = models.CharField(
```

Comment:
No, that makes sense.  It's assumed that this is always a CSS file, so there's no need for the extension. I like ``css_slug``.